kythe/go/platform/tools/entrystream/entrystream.go

```diff
@@ -55,8 +59,11 @@ type entrySet struct {
 }
 
 var (
-	readJSON        = flag.Bool("read_json", false, "Assume stdin is a stream of JSON entries instead of protobufs")
-	writeJSON       = flag.Bool("write_json", false, "Print JSON stream as output")
+	readJSON     = flag.Bool("read_json", false, "Assume stdin is a stream of JSON entries instead of protobufs")
+	writeJSON    = flag.Bool("write_json", false, "Print JSON stream as output")
```

Comment:
Can you either add input validation to make sure conflicting options aren't set (readJSON && readRiegeli) or switch to an enum since there are 3 options now?

Reply:
Switched to enum